Data/Csv/Conversion.hs

```diff
@@ -489,7 +492,7 @@ class ToField a where
     toField :: a -> Field
 
 -- | 'Nothing' if the 'Field' is 'B.empty', 'Just' otherwise.
-instance FromField a => FromField (Maybe a) where
+instance {-# OVERLAPPABLE #-} FromField a => FromField (Maybe a) where
```

Comment:
Added an `#ifdef`. The argument could be made of course that it is `music-parts` that should be fixed since it defines a `FromField (Maybe Foo)` instance.